src/units/tetris/classic-tetris.js

```diff
@@ -713,7 +713,7 @@ class ClassicTetris {
 		this.gameOverLine = -1;
 
 		// frames until the piece automatically moves down
-		this.framesTilDrop = 36 + this._getFramesPerGridcell(this.level); // 18 + this._getFramesPerGridcell(this.level);
+		this.framesTilDrop = 35 + this._getFramesPerGridcell(this.level); // 18 + this._getFramesPerGridcell(this.level);
```

Comment:
Could you explain how this affects performance?

Reply:
Makes it the tiniest bit faster, likely not noticed by human eye, but can be changed.